docs/content/contributing/bicep/bicep-contribution-flow/_index.md

```diff
@@ -193,7 +278,53 @@ To use the environment's pipelines you should use the information you gathered d
 
 {{< /expand >}}
 
-<p>
+
+#### 3.1.2 Authentication secrets
+
+In addition to shared repository secrets detailed above, additional GitHub secrets are required to allow the deployment identity authentication to Azure.
+
+Expand and follow the option corresponding to the deployment identity setup chosen at [Step 2](#2-configure-a-deployment-identity-in-azure) and use the information you gathered during that step.
+
+
+{{< expand "➕ Option 1 [Recommended]: Authenticate via OIDC" "expand/collapse" >}}
+
+Create the following environment secrets in the `avm-validation` GitHub environment created at [Step 1](#1-fork-the-module-source-repository)
```

Comment:
Depending on the answer to [this comment](https://github.com/Azure/Azure-Verified-Modules/pull/1773/files#r1898998266), the environment may or may not exist...